de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537590_sys_gh5856_Add_ExternalOrderId_toC_Payment_and_C_order.sql

```diff
@@ -0,0 +1,125 @@
+-- 2019-12-02T13:27:59.281Z
```

Comment:
please find a nice spot in the sales-C_Order and C_Payment windows "Advanced Edit" area and display them there (read-only)